### PR TITLE
Fixed missing 18.03.1-ee-3 and ee-4 release notes

### PR DESCRIPTION
**Pull request by ahh-docker**

### CODE REVIEW

**Comment by ahh-docker** [comment]
@andrewhsu Thanks, I'll fix these before we have @L-Hudson look over them.

**Comment by ahh-docker** [comment]
@andrewhsu Are there any other release notes I need to get added? Let me know, and I'll fix them. Thanks for your help!

File: engine/release-notes.md [hunk 1]
```diff
@@ -203,6 +203,59 @@ Ubuntu 14.04 "Trusty Tahr" [docker-ce-packaging#255](https://github.com/docker/d
 
 ## Older Docker Engine EE Release notes
 
+### 18.03.1-ee-4 
+2018-10-25
+
+> *** NOTE: **** If you're deploying UCP or DTR, use Docker EE Engine 17.06.
```
Comment (thaJeztah) [inline engine/release-notes.md]: EE 18.09 or 17.06 I guess (I think we have a snippet somewhere explaining that 18.03 is a standalone ("engine") release.

(probably has to be mentioned in the other patch release below as well)

Comment (ahh-docker) [reply engine/release-notes.md]: @thaJeztah fixed. Ok to merge?

File: engine/release-notes.md [hunk 2]
```diff
@@ -203,6 +203,59 @@ Ubuntu 14.04 "Trusty Tahr" [docker-ce-packaging#255](https://github.com/docker/d
 
 ## Older Docker Engine EE Release notes
 
+### 18.03.1-ee-4 
+2018-10-25
+
+> *** NOTE: *** If you're deploying UCP or DTR, use Docker EE Engine 18.09 or higher. 18.03 is an engine only release.
```
Comment (thaJeztah) [inline engine/release-notes.md]: oh; hold: there should be no space between the `***` otherwise it doesn't render as "bold/italic"

**Pull request merged**